libs/output-mapping/tests/Writer/Workspace/WriterWorkspaceTest.php

```diff
@@ -112,6 +115,130 @@ public function testSnowflakeTableOutputMapping(): void
         ]);
     }
 
+    protected function prepareWorkspaceWithTablesWithTimestampColumn(
+        string $bucketBackend,
+        string $bucketName,
+        string $tablePrefix = ''
+    ): void {
+        $temp = new Temp();
+        $root = $temp->getTmpFolder();
+        $bucketId = $this->clientWrapper->getBasicClient()->createBucket(
+            name: $bucketName,
+            stage: 'in',
+            backend: $bucketBackend
+        );
+        // Create tables
+        $csv1a = new CsvFile($root . DIRECTORY_SEPARATOR . 'table1a.csv');
+        $csv1a->writeRow(['id', 'name', '_timestamp']);
```

Comment:
Ten timestamp tu je zbytečný ne? KBC to webalizuje, takže se to naimportuje jako `timestamp` sloupec.

Reply:
smazané